testing/xsel/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: dai9ah <dai9ah@protonmail.com>
+# Maintainer:
+pkgname=xsel
+pkgver=1.2.0
+pkgrel=0
+pkgdesc="Command-line program for manipulating the X selection"
+url="http://www.vergenet.net/~conrad/software/xsel"
+arch="all"
+license="custom"
```

Comment:
[COPYING](https://github.com/kfish/xsel/blob/master/COPYING) is clearly [MIT](https://fedoraproject.org/wiki/Licensing:MIT).